src/azul/chalice.py

```diff
@@ -189,15 +189,30 @@ def _api_gateway_context_middleware(self, event, get_response):
         finally:
             config.lambda_is_handling_api_gateway_request = False
 
+    hsts_max_age = 60 * 60 * 24 * 365 * 2
+
+    response_headers = {
+        'Content-Security-Policy': "default-src 'self'",
+        'X-Content-Type-Options': 'nosniff',
+        'X-Frame-Options': 'DENY',
+        'Referrer-Policy': 'strict-origin-when-cross-origin',
+        'Strict-Transport-Security': f'max-age={hsts_max_age};'
+                                     ' includeSubDomains;'
+                                     ' preload',
+        'X-XSS-Protection': '1; mode=block'
```

Comment:
This should use the string functions for joining words and so on that you added in some other PR, otherwise we end up with inconsistencies again.